binary_transparency/firmware/devices/usb_armory/bootloader/ext4.go

```diff
@@ -28,6 +28,21 @@ type Partition struct {
 	_offset int64
 }
 
+func (d *Partition) GetPartitionSize() (uint64, error) {
```

Comment:
Will be good to add a comment on GetPartitionSize(), what is the intent, which partition?

Reply:
Done, also changed the name to better highlight that it's the ext4 fs size.